spec/kenji_spec.rb

```diff
@@ -40,7 +37,8 @@ def app; app_for('1'); end
 
     it 'should return 500 for exceptions' do
       get '/main/crasher'
-      expected_response = {status: 500, message: 'Something went wrong...'}.to_json
+      expected_response = {status: 500,
+                           message: 'Something went wrong: kaboom!'}.to_json
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.